docs/src/main/sphinx/connector/snowflake.md

```diff
@@ -0,0 +1,96 @@
+# Snowflake connector
+
+```{raw} html
+<img src="../_static/img/snowflake.png" class="connector-logo">
+```
+
+The Snowflake connector allows querying and creating tables in an
+external [Snowflake](https://www.snowflake.com/) account. This can be used to join data between
+different systems like Snowflake and Hive, or between two different
+Snowflake accounts.
+
+## Configuration
+
+To configure the Snowflake connector, create a catalog properties file
+in `etc/catalog` named, for example, `example.properties`, to
+mount the Snowflake connector as the `snowflake` catalog.
+Create the file with the following contents, replacing the
+connection properties as appropriate for your setup:
+
+```none
+connector.name=snowflake
+connection-url=jdbc:snowflake://<account>.snowflakecomputing.com
+connection-user=root
+connection-password=secret
+snowflake.account=account
+snowflake.database=database
+snowflake.role=role
+snowflake.warehouse=warehouse
+```
+
+### Arrow serialization support
+
+This is an experimental feature which introduces support for using Apache Arrow
+as the serialization format when reading from Snowflake.  Please note there are
+a few caveats:
+
+- Using Apache Arrow serialization is disabled by default. In order to enable
+  it,  add `--add-opens=java.base/java.nio=ALL-UNNAMED` to the Trino
+  {ref}`jvm-config`.
+
+### Multiple Snowflake databases or accounts
+
+The Snowflake connector can only access a single database within
+a Snowflake account. Thus, if you have multiple Snowflake databases,
+or want to connect to multiple Snowflake accounts, you must configure
+multiple instances of the Snowflake connector.
+
+% snowflake-type-mapping:
+
+## Type mapping
```

Comment:
Type mapping should be separated into "Snowflake type to Trino type mapping" & "Trino type mapping to Snowflake type mapping"